lib/requester/requester.js

```diff
@@ -94,7 +97,12 @@ Requester.prototype.request = function (item, cb, scope) {
         startTime = Date.now();
 
     return requests(request, requestOptions, function (err, res, resBody) {
-        if (err) { return cb.call(scope || this, err, undefined, request); }
+        if (err) {
+            // bubble up http errors
+            // @todo - Should we send an empty sdk Response here?
+            triggers.io(err, cursor, HTTP, undefined, request);
```

Comment:
@czardoz we cannot always send this as type. we need to define accurate type info that is to be sent during requesting.